test/kata3.numberToReversedDigits.test.js

```diff
@@ -1,5 +1,7 @@
 const { numberToReversedDigits } = require("../src");
 
 describe("numberToReversedDigits", () => {
-  test("returns a reversed array of the number's digits", () => {});
+  test("returns a reversed array of the number's digits", () => {
+    expect(numberToReversedDigits([1,2,3])).toEqual([3,2,1]);
```

Comment:
For better readability I would add a space after each array element: [1, 2, 3], [3, 2, 1]